pyscriptjs/tests/unit/pyconfig.test.ts

```diff
@@ -11,25 +13,43 @@ const covfefeConfig = {
             lang: 'covfefe',
         },
     ],
-    wonerful: 'discgrace',
+    wonderful: 'disgrace',
```

Comment:
@antocuni these were deliberately misspelled, since it is the `covfefe` language aka it is inspired by typos made by `Donald Trump`

Reply:
I don't think we should have this kind of words in the project 👍 
also
>Donald Trump

Reply:
ah, I actually wondered why you did so many mistakes :).

Reply:
Yes, it was just an easter egg -- was trying to be funny haha -- but sure you can remove it if you want lol

Reply:
it's not a big deal, but typos are subtle: I can imagine that in the future someone might write a test which involves `wonderful` and `disgrace` (with the correctly spelling) and then spend several minutes trying to understand whether the test doesn't work, only to understand later that it was because `wonerful != wonderful`.

This scenario is not only hypothetical: it happened to me for real in the past, because of actual unwanted typos :man_facepalming: . No need to introduce deliberate ones :)